app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def random_greeting
+    greetings = ['Hello', 'Hi', 'Hola', 'Bonjour', 'Guten Tag']
+    render json: { greeting: greetings.sample }
+  end
```

Comment:
- [ ] As mentioned by the previous reviewer, it would be nice fetch the greeting messages from the greetings table to follow the project requirement(see image :point_down:)
![Screenshot from 2024-01-18 19-15-19](https://github.com/obibaadoma/Hello-World-Webpack/assets/91142432/52cce9ab-1145-45cd-aeb0-38b44e303a31)

Reply:
Done, when localhost/random_greetings isloading all the different greetings are loaded.